includes/class-wc-privacy-erasers.php

```diff
@@ -0,0 +1,315 @@
+<?php
+/**
+ * Personal data erasers.
+ *
+ * @since 3.4.0
+ * @package WooCommerce\Classes
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Privacy_Erasers Class.
+ */
+class WC_Privacy_Erasers {
+	/**
+	 * Finds and erases customer data by email address.
+	 *
+	 * @since 3.4.0
+	 * @param string $email_address The user email address.
+	 * @param int    $page  Page.
+	 * @return array An array of personal data in name value pairs
+	 */
+	public static function customer_data_eraser( $email_address, $page ) {
+		$response = array(
+			'items_removed'  => false,
+			'items_retained' => false,
+			'messages'       => array(),
+			'done'           => true,
+		);
```

Comment:
I was not able to get the Remove Personal Data tool to work successfully. I get the following error every time:
`{"success":false,"data":"Error: Expected num_items_removed key in response array from Customer Data eraser (index 2)."}`

[It seems that the correct keys are num_items_removed and num_items_retained](https://github.com/WordPress/WordPress/blob/master/wp-admin/includes/ajax-actions.php#L4586-L4603), and they expect numbers. Maybe we're on different versions of the WP branch? I tested with the latest SVN-synced version from GitHub.

Reply:
https://core.trac.wordpress.org/ticket/43602#comment:44